.github/workflows/build.yml

```diff
@@ -15,10 +15,11 @@ jobs:
     steps:
       - name: checkout code
         uses: actions/checkout@v2
-      - name: Set up JDK 11
-        uses: actions/setup-java@v1
+      - name: Set up JDK 17
+        uses: actions/setup-java@v4
         with:
-          java-version: 11
+          distribution: 'temurin'
```

Comment:
I have my doubts with this distribution. it might not cause any issue but I would prefer to use the same we use in OSS: https://github.com/spinnaker/kork/blob/master/.github/workflows/release.yml#L43